tools/vagrant/simple/resume_build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
should the script above just call this script so there aren't two copies to maintain?